Added Steps for checking if node and npm is installed. (#246)
oppia/oppiabot: README.md

2. Install [Node.Js](https://nodejs.org/) and [NPM](https://www.npmjs.com/) on your machine using the following commands:

    First check if you already have Node.js and NPM using:
    ```bash
    node -v
    npm -v
    ```

>     If these commands show some versions, then you can jump to Point No. 3, else proceed in this point:

Reviewer: ```suggestion
    If these commands show some versions, then you can jump to Point No. 3, else proceed with these commands:
```

Author: Sure sir, Changing them right away.
